### PR TITLE
LPS-169427 | Test Fix| Master

### PR DESCRIPTION
**Pull request by evanilson-santana**
https://issues.liferay.com/browse/LPS-169427

### CODE REVIEW

**Comment by evanilson-santana** [comment]
For this test case, we need to check if ERC field can use on the Expression Builder. I changed the old validation to a new one. The old validation was `Integer > External Reference Code` that no sense. The new is `customObjectField == externalReferenceCode` were customObjectField is a string field.  This makes sense because ERC field is a string field.

cc: @timpak

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 1]
```diff
@@ -1654,22 +1652,12 @@ definition {
...
-
-			Navigator.gotoBack();
-
-			ObjectPortlet.addCustomFieldEntryViaUI(
-				customField = "Custom Integer",
-				entry = "2147483647");
-
-			ObjectAdmin.goToCustomObject(objectName = "CustomObject162179");
-
-			ObjectPortlet.viewEntry(entry = "2147483647");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:1672]: Hey @evanilson-santana,

Add a correct entry with a string field. You need to confirm the correct entry for more precision on testcase.

**Review by andreefarias — CHANGES_REQUESTED** [review]
Hey @evanilson-santana,

Check my comment and fix the testcase, please. Thanks!

**Comment by andreefarias** [comment]
ci:forward

**Pull request closed without merging**